### PR TITLE
Add action for getting information on stacks

### PR DESCRIPTION
**Pull request by mhahn**
Currently just logs the outputs.

### CODE REVIEW

File: stacker/actions/info.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+class Action(BaseAction):
+    """Get information on CloudFormation stacks.
+
+    Displays the outputs for the set of CloudFormation stacks.
+
+    """
+
+    def run(self, *args, **kwargs):
+        logger.info('Outputs for stacks: %s', self.context.get_fqn())
```
Comment (phobologic) [inline stacker/actions/info.py]: Does this actually work?  Doesn't get_fqn need a stack name?

File: stacker/actions/info.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        for stack in self.context.get_stacks():
+            provider_stack = self.provider.get_stack(stack.fqn)
```
Comment (phobologic) [inline stacker/actions/info.py]: This will raise a StackDoesNotExist exception.

File: stacker/commands/stacker/info.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+
+class Info(StackerCommand):
+
+    name = 'info'
+    description = __doc__
+
+    def run(self, options, **kwargs):
+        super(Info, self).run(options, **kwargs)
+        action = info.Action(options.context, provider=options.provider)
```
Comment (phobologic) [inline stacker/commands/stacker/info.py]: Probably worth adding an argument to get the output from a single stack if specified.

Comment (phobologic) [inline stacker/commands/stacker/info.py]: Also, might be cool to have a --output or --format argument.  Maybe not necessary here, but I could see someone wanting this in a machine readable format (like csv).

File: stacker/commands/stacker/info.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
```
Comment (phobologic) [inline stacker/commands/stacker/info.py]: Does Info sound right to you?  Maybe just Outputs for now or something like that?

Comment (mhahn) [inline stacker/commands/stacker/info.py]: Yea, I just wasn't sure if we wanted to put more information in their like
current status or something

File: stacker/actions/info.py (hunk 1 above)
Comment (mhahn) [inline stacker/actions/info.py]: this defaults to simply the namespace

File: stacker/commands/stacker/info.py (hunk 4 above)
Comment (mhahn) [inline stacker/commands/stacker/info.py]: maybe including like:

- created date
- last changed date

would be useful and fit more with info?

haven't looked into whether or not we have access to that info

Comment (phobologic) [inline stacker/commands/stacker/info.py]: Yeah, that's fine.  If you think there's more info we can provide, lets leave it as Info - doesn't necessarily need to happen in this PR.

File: stacker/commands/stacker/info.py (hunk 3 above)
Comment (mhahn) [inline stacker/commands/stacker/info.py]: lets look into the --output or --format in the next revision

**Pull request merged**